docs/openapi/pn-address-book-api-internal.yaml

```diff
@@ -329,6 +338,9 @@ components:
       properties:
         value:
           type: string
+          maxLength: 50
+          #Tel. number or email address
+          pattern: ^(((00|\+)393\d{8,9})|([\p{L}0-9!#\$%*\/?\|\^\{\}`~&'+\-=_]+(?:[.-][\p{L}0-9!#\$%*\/?\|\^\{\}`~&'+\-=_]+){0,10}@\w+(?:[.-]\w+){0,10}\.\w{2,10}))$
```

Comment:
Su sto punto, mi è venuto un dubbio, che riguarda l'app IO.
in pratica, quando l'utente attiva IO, mi sa che il FE ci manda una scritta (non ricordo il valore però), perchè appunto il value era obbligatorio. Il valore non è poi considerato dal BE. Riesci a fare un check con quelli del FE per capire cosa mandano, e allineare poi questa regexpt di conseguenza?

Reply:
Resolved with 
commit 408ce9e09e24000b4f8705be48c8f0be4ca3f001